jbmc/src/jdiff/jdiff_parse_options.cpp

```diff
@@ -321,7 +321,12 @@ bool jdiff_parse_optionst::process_goto_program(
     // instrument cover goals
     if(cmdline.isset("cover"))
     {
-      if(instrument_cover_goals(options, goto_model, get_message_handler()))
+      const auto cover_config = get_cover_config(
+        options, goto_model.symbol_table, log.get_message_handler());
```

Comment:
:question: Why do the two calls use different message handlers?

Reply:
copy-paste mistake, good spot!